mm2src/coins/utxo/pb.rs

```diff
@@ -1,92 +1,108 @@
-// RPC MESSAGES
-
+#[allow(clippy::derive_partial_eq_without_eq)]
 #[derive(Clone, PartialEq, ::prost::Message)]
-pub struct GetMempoolInfoRequest {
-}
+pub struct GetMempoolInfoRequest {}
+#[allow(clippy::derive_partial_eq_without_eq)]
```

Comment:
This file is auto-generated, the new lib version does this in favor of implementing `Eq`.
Maybe to save space?

Reply:
> Maybe to save space?

Related issue and comment https://github.com/tokio-rs/prost/issues/661#issuecomment-1150140898 , maybe because floating numbers don't implement `Eq` so to fix this, they need to derive `Eq` depending on the data types in the message which might be complicated for them. This is just my guess as well :)

We can leave this as is then.